setup.py

```diff
@@ -20,7 +20,10 @@
         'Flask>=0.12',
         'flask-compress',
         'plotly',
-        'dash_renderer',
+        'dash_renderer==0.17.0',
+        'dash-core-components==0.43.0',
+        'dash-html-components==0.13.5',
+        'dash-table==3.3.0'
```

Comment:
Until we implement the tip-o-master cross-package testing protocol, do you want to update `.circleci/requirements/*` to match these versions?